app/controllers/spree/reviews_controller.rb

```diff
@@ -36,6 +44,25 @@ def create
     end
   end
 
+  def update
+    params[:review][:rating].sub!(/\s*[^0-9]*\z/, '') if params[:review][:rating].present?
```

Comment:
Would you mind changing any reference of `params[:review][:ATTRIBUTE]` to use strong params with `review_params[:ATTRIBUTE]`